ext/snowcrash/src/Just.h

```diff
@@ -0,0 +1,48 @@
+//
+//  Just.h
+//  snowcrash
+//
+//  Created by Thomas Jandecka on 11/7/19.
+//  Copyright (c) 2019 Apiary Inc. All rights reserved.
+//
+
+#ifndef SNOWCRASH_JUST_H
+#define SNOWCRASH_JUST_H
+
+#include <utility>
+
+namespace mson
```

Comment:
Do you mean, `mson` ns is good place for `just`? I mean it is generalized concept so it should be in some  `util` ns or something similar.